02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,108 @@
+html {
+    background-image: url("https://wallpaperscraft.com/image/paper_crumpled_light_shadow_35117_1920x1080.jpg");
```

Comment:
Thank you so much! 
I actually wanted to ask you about your app that you made with git, the ACA logo at the top has a moving component to it, what is that? A video? Jquerry? Just want to get better at making a living website


 
 
  On Wed, Jul 27, 2016 at 2:21 PM, Roberto C Navarro<notifications@github.com> wrote:   
In 02LessonTwo/homework/css/style.css:
> @@ -0,0 +1,108 @@
> +html {
> +    background-image: url("https://wallpaperscraft.com/image/paper_crumpled_light_shadow_35117_1920x1080.jpg");

I like this! Great job, you are showing me you have creativity and that is what it takes to become a top notch web developer.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.